aws-codeguruprofiler-profilinggroup/src/main/java/software/amazon/codeguruprofiler/profilinggroup/UpdateHandler.java

```diff
@@ -58,6 +61,9 @@
                 logger.log(String.format("Policy for [%s] for accountId [%s] has been successfully removed!", profilingGroupName, awsAccountId));
             }
 
+            // Delete existing Notification Channels when update is called
+            deleteNotificationChannels(profilingGroupName, proxy);
```

Comment:
Is this correct? If something is updated in the profiling group, all the notifications are deleted? What if I added a new ARN in the template? Instead of adding an ARN all of them will be removed...